src/reference-designs/tko-on-vsphere.md

```diff
@@ -446,13 +446,13 @@ VMware recommends that you deploy Tanzu Kubernetes Grid on non-stretched vSphere
 
 ## Container Registry
 
-The Tanzu Editions include [Harbor](https://goharbor.io/) as a container registry. Harbor provides a location for pushing, pulling, storing, and scanning container images to be used in your Kubernetes clusters. There are three main supported installation methods for Harbor:
+The Tanzu Editions include [Harbor](https://goharbor.io/) as a container registry. Harbor provides a location for pushing, pulling, storing, and scanning container images to be used in your Kubernetes clusters. There are three officially documented installation methods for Harbor:
 
 * **[Tanzu Kubernetes Grid Package deployment](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.4/vmware-tanzu-kubernetes-grid-14/GUID-packages-harbor-registry.html)** to a Tanzu Kubernetes Grid cluster - this installation method is recommended for general use cases.
 
-* **[Helm-based deployment](https://goharbor.io/docs/2.1.0/install-config/harbor-ha-helm/)** to a Kubernetes cluster - this installation method may be preferred for customers already invested in Helm.
+* **[Helm-based deployment](https://goharbor.io/docs/2.1.0/install-config/harbor-ha-helm/)** to a Kubernetes cluster - this installation method may be preferred for customers already invested in Helm. Helm deployments are only supported by the Open Source community and not by VMware Global Support Services.
 
-* **[VM-based deployment](https://goharbor.io/docs/2.1.0/install-config/installation-prereqs/)** using docker-compose - VMware recommends using this installation method in cases where Tanzu Kubernetes Grid is being installed in an air-gapped or Internet-less environment and no pre-existing Kubernetes clusters exist on which to install Harbor. When Kubernetes is already available, the Helm based deployment also works for air-gapped environments.
+* **[VM-based deployment](https://goharbor.io/docs/2.1.0/install-config/installation-prereqs/)** using docker-compose - VMware recommends using this installation method in cases where Tanzu Kubernetes Grid is being installed in an air-gapped or Internet-less environment and no pre-existing image registry exists to host the TKG system images. When Kubernetes is already available, the Helm based deployment also works for air-gapped environments. VM-based deployments are only supported by VMware Global Support Services to host the system images for air-gapped or internet-less deployments and must not be used for hosting application images.
```

Comment:
...`docker-compose`... exists to host the Tanzu Kubernetes Grid system images. In an air-gapped or Internet-less deployment, VMware Global Support Services only supports hosting system images on VM-based deployments. Hosting application images is not supported.

Reply:
This should be addressed in the latest push